package.json

```diff
@@ -79,7 +80,8 @@
     "format:check": "prettier --check \"**/*.{ts,tsx,json,scss,css}\"",
     "typecheck": "tsc --project tsconfig.json --noEmit",
     "prepare": "husky install",
-    "jest-preview": "jest-preview"
+    "jest-preview": "jest-preview",
+    "update-toc": "node update-toc.js"
```

Comment:
This script should not be in the root directory. We have a scripts directory referenced elsewhere in this file that should be used.

Reply:
Fixes.